packages/gatsby/src/bootstrap/load-plugins/load.js

```diff
@@ -130,12 +132,7 @@ module.exports = (config = {}) => {
         // Make sure key is unique to plugin options. E.g there could
         // be multiple source-filesystem plugins, with different names
         // (docs, blogs).
-        id: createNodeId(
-          plugin.options
-            ? plugin.name + JSON.stringify(plugin.options)
```

Comment:
Is `plugin.name` undefined here? Was it intended to use `info.name` or `plugin.resolve`?

Reply:
I think it should be `info.name`